### PR TITLE
[4.0] Added box-shadow to user dropdown menu

### PR DESCRIPTION
**Pull request by hardik-codes**
Pull Request for Issue #23791 

### Summary of Changes
Added ```box-shadow:0px 8px 10px #C0C0C0, -10px 8px 15px #C0C0C0, 10px 8px 15px #C0C0C0;``` to   the ```.dropdown-menu``` selector in the file ```administrator/templates/atum/scss/blocks/_header.scss```

### Testing Instructions
Open the user dropdown menu

### Expected result
boxshadow

### Actual result
boxoriginal

### Documentation Changes Required
None

### CODE REVIEW

**Comment by hardik-codes** [comment]
@C-Lodder please review the PR

**Comment by ciar4n** [comment]
To keep shadows consistent, use the available box-shadow variable... https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/templates/atum/scss/_variables.scss#L62

**Comment by hardik-codes** [comment]
But the box-shadow variable cannot be used as it is because of shadows on 3 sides so instead of using the variable may I use the values directly for consistency?

**Comment by ciar4n** [comment]
If you wish you can create a new dropdown box-shadow variable if you feel the current variable is not suitable. Up to you but I would consider `'0px 8px 10px #C0C0C0, -10px 8px 15px #C0C0C0, 10px 8px 15px #C0C0C0;'` excessive.

**Comment by brianteeman** [comment]
If I read this correctly then this is not correct for RTL

**Comment by hardik-codes** [comment]
@brianteeman apart from the box-shadow on 3 sides what else could be done so that it is correct for RTL

**Comment by brianteeman** [comment]
The three sides are different in rtl

**Comment by hardik-codes** [comment]
Different in the widths?

**Comment by C-Lodder** [comment]
@brianteeman - When he says 3 sides, he means left/right/bottom and this works perfectly fine on RTL.

@hardik-codes - Using the variable mentioned by @ciar4n is fine

**Comment by C-Lodder** [comment]
@hardik-codes - Using the variable mentioned by @ciar4n is perfectly fine

**Comment by hardik-codes** [comment]
Ok @C-Lodder

**Comment by hardik-codes** [comment]
I hope its now fine @C-Lodder

File: administrator/templates/atum/scss/blocks/_header.scss [hunk 1]
```diff
@@ -58,6 +58,7 @@
     padding: 0;
     border: 0;
     z-index: 1041;
+    box-shadow:$atum-box-shadow;
```
Comment (C-Lodder) [inline administrator/templates/atum/scss/blocks/_header.scss]: Add a space after the colon (`:`)

**Review by C-Lodder — APPROVED** [review]

**Comment by hardik-codes** [comment]
@C-Lodder I accidentally typed 'before' instead of 'after' in the commit message how can I fix that?
Should I delete this commit and generate a new commit.

**Comment by hardik-codes** [comment]
Changed the commit message successfully !!

**Comment by brianteeman** [comment]
Sorry for the confusion

**Comment by astridx** [comment]
I applied this path via `git fetch origin pull/23833/head:boxShadow` and run `npm install`.
Before:
control panel rsfd administration 1
After:
control panel rsfd administration

**Comment by astridx** [comment]
I have tested this item :white_check_mark: successfully on 23055b010c2405752c488aa3495ddd949da64bd6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23833">issues.joomla.org/tracker/joomla-cms/23833</a>.</sub>

**Comment by hardik-codes** [comment]
Members please test this PR @C-Lodder

**Comment by C-Lodder** [comment]
@wilsonge

**Comment by hardik-codes** [comment]
One more tester please @Quy

**Comment by Quy** [comment]
There is a visible outline without this PR.

shadow

**Comment by Quy** [comment]
Here is the backend of the new design. Shadow is no longer an issue. I recommend closing this PR.

shadow

**Comment by wilsonge** [comment]
Closing per @Quy 's comment

**Pull request closed without merging**